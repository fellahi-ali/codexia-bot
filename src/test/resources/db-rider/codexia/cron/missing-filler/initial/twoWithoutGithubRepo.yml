codexia_project:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 1662
        coordinates: "instaloader/instaloader"
        author: "iakunin"
        deleted: null
        project_created_at: "2020-05-05 10:29:21"
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 1999
        coordinates: "arpit9667/algorithms-js"
        author: "iakunin"
        deleted: null
        project_created_at: "2020-05-07 11:06:40"
