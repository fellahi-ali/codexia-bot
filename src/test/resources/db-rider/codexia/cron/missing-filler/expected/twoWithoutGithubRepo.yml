codexia_project:
    -
        external_id: 1662
        coordinates: "instaloader/instaloader"
        author: "iakunin"
        deleted: null
        project_created_at: "2020-05-05 10:29:21"
    -
        external_id: 1999
        coordinates: "arpit9667/algorithms-js"
        author: "iakunin"
        deleted: null
        project_created_at: "2020-05-07 11:06:40"

github_repo:
    -
        external_id: "61200117"
        full_name: "instaloader/instaloader"
        html_url: "https://github.com/instaloader/instaloader"
    -
        external_id: "253000011"
        full_name: "arpit9667/algorithms-js"
        html_url: "https://github.com/arpit9667/algorithms-js"

github_repo_source:
    -
        source: "CODEXIA"
        external_id: "1662"
    -
        source: "CODEXIA"
        external_id: "1999"
